### PR TITLE
zmarkdown: escape chars in titles and author

### PR DESCRIPTION
**Pull request by artragis**

### CODE REVIEW

**Comment by artragis** [comment]
ping @vhf

**Comment by artragis** [comment]
@sandhose fixed

**Comment by artragis** [comment]
@vhf : test is here. and I hope my title is now as you expect.

File: packages/zmarkdown/__tests__/server.tests.js [hunk 1]
```diff
@@ -132,6 +132,16 @@ describe('LaTeX endpoint', () => {
     expect(metadata.disableToc).toBe(false)
   })
 
+  it('escapes title and author', async () => {
+    const response = await a.post(latex,
```
Comment (vhf) [inline packages/zmarkdown/__tests__/server.tests.js]: Take a look at what you commit. The snapshot shows that you are not testing your changes. I guess the endpoint used on this line is not the right one.

Comment (artragis) [reply packages/zmarkdown/__tests__/server.tests.js]: you seem right.

**Comment by artragis** [comment]
Seems ok now.

**Comment by artragis** [comment]
thanks @vhf

**Pull request merged**